tests/system/admin_overlay/conftest.py

```diff
@@ -0,0 +1,26 @@
+from google.cloud.client import ClientWithProject
+
+import pytest
+
+
+INSTANCE_PREFIX = "admin-overlay-instance"
+BACKUP_PREFIX = "admin-overlay-backup"
+ROW_PREFIX = "test-row"
+
+DEFAULT_CLUSTER_LOCATIONS = ["us-east1-b"]
+REPLICATION_CLUSTER_LOCATIONS = ["us-east1-b", "us-west1-b"]
+TEST_TABLE_NAME = "system-test-table"
+TEST_BACKUP_TABLE_NAME = "system-test-backup-table"
+TEST_COLUMMN_FAMILY_NAME = "test-column"
+TEST_COLUMN_NAME = "value"
+NUM_ROWS = 500
+INITIAL_CELL_VALUE = "Hello"
+NEW_CELL_VALUE = "World"
+
+
+@pytest.fixture(scope="session")
+def admin_overlay_project_id():
+    # Instantiate a meaningless client to get project name
+    # from environment variables.
+    client = ClientWithProject()
+    yield client.project
```

Comment:
Creating a client just for this seems a bit messy. 

Looking through the ClientWithProject code, [it seems like it just uses](https://github.com/googleapis/python-cloud-core/blob/78ce8a6f7c781cf2585131d7961beec9069a206c/google/cloud/_helpers/__init__.py#L152C22-L152C43) `google.auth.default()` for this

I think you should also check `os.getenv("GOOGLE_CLOUD_PROJECT")` first though, so we can configure a different project if needed (And maybe put this up with the constants as ADMIN_TEST_PROJECT or something so it's discoverable?)